app/uk/gov/hmrc/fhregistrationfrontend/views/forms/company_registration_number.scala.html

```diff
@@ -47,39 +46,69 @@
 @title = @{
   Messages("fh.companyRegistrationNumber.label")
 }
+@backLink(navigation: Navigation) = @{
+    navigation.back.map {
```

Comment:
Should this code be here or in the Navigation file?  It seems like a lot to have to write all of this in on pages that use a backlink.

Reply:
I have mentioned a note here that This will all be removed and will refer to Back.scala.html file once rebased

Reply:
@ginnyamazed please let me know if it's ok to resolve this conversation and approve the PR?

Reply:
good to go